src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -55847,16 +56204,17 @@ class Outer
         if (x0 == null) return;
         M2(x0) = y0;
         x0.ToString();
+        M2(x0).ToString();
```

Comment:
>M2(x0).ToString(); [](start = 8, length = 18)

The line above actually changes the flow sate of ```x0```, consider placing this line above it, so that both ```M2(x0)```` calls are based on the same state.

Reply:
I don't mind changing those, but I think de-reference changes the flow state iff it produces a warning.

---
In reply to: [244063601](https://github.com/dotnet/roslyn/pull/31813#discussion_r244063601) [](ancestors = 244063601)